aviary/interface/methods_for_level2.py

```diff
@@ -2436,31 +2436,46 @@ def _add_gasp_landing_systems(self):
         )
 
     def _add_fuel_reserve_component(self, reserves_name=Mission.Design.RESERVE_FUEL):
-        reserves_val = self.aviary_inputs.get_val(Aircraft.Design.RESERVES)
-        if reserves_val <= 0:
-            reserves_val = -reserves_val
-            self.model.add_subsystem(
-                "reserves_calc",
-                om.ExecComp(
-                    f"reserve_fuel = {reserves_val}*(takeoff_mass - final_mass)",
-                    takeoff_mass={"units": "lbm"},
-                    final_mass={"units": "lbm"},
-                    reserve_fuel={"units": "lbm"}
-                ),
-                promotes_inputs=[
-                    ("takeoff_mass", Mission.Summary.GROSS_MASS),
-                    ("final_mass", Mission.Landing.TOUCHDOWN_MASS),
-                ],
-                promotes_outputs=[("reserve_fuel", reserves_name)],
-            )
-        elif reserves_val > 10:
-            self.model.add_subsystem(
-                "reserves_calc",
-                om.ExecComp(
-                    f"reserve_fuel = {reserves_val}",
-                    reserve_fuel={"val": reserves_val, "units": "lbm"}
-                ),
-                promotes_outputs=[("reserve_fuel", reserves_name)],
-            )
+        reserves_val = self.aviary_inputs.get_val(Aircraft.Design.RESERVES, units='lbm')
+        reserves_fac = self.aviary_inputs.get_val(
```

Comment:
this should be "frac" or "fraction", "fac" sounds like it is a factor that will be applied to the reserves.

Reply:
renamed "frac" to "frac".